translation.rst

```diff
@@ -386,6 +388,9 @@ The ``translation:update`` command looks for missing translations in:
   :ref:`twig.paths <config-twig-paths>` config options);
 * Any PHP file/class that injects or :doc:`autowires </service_container/autowiring>`
   the ``translator`` service and makes calls to the ``trans()`` method.
+* Any PHP file/class stored in the ``src/`` directory that creates a
+  translatable object using the constructor or the ``t()`` method or calls
```

Comment:
Did you forget to link to the section about "translatable object"?

Reply:
Link added